<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>account</class>
 <widget class="QWidget" name="account">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="account_inquirelineEdit">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>0</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="account_inquireButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="account_deleteButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>0</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="account_quitButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>0</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>月总览</string>
   </property>
  </widget>
  <widget class="QPushButton" name="account_saveButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="account_yearButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>50</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>年总览</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="account_monthlineEdit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>0</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="account_yearlineEdit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="account_addButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="account_moneylineEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="account_textlineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="account_datelineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
